adobe/spectrum-web-components
# chore: take latest CSS dependencies

## Description
Take CSS update the sequesters `:hover` styles within the `@media (hover: hover) {}` media query, which prevents it in the _majority_ of mobile/touch-only contexts.

## Related issue(s)
- fixes #2598

## How has this been tested?
    1. Go [here](https://spectrum-css--spectrum-web-components.netlify.app/storybook/index.html?path=/story/menu--default)
    2. In a mobile or touch-only context (can be triggered with Chrome Dev Tools)
    3. Click on of the Menu Items
    4. See that the "hover" state is not triggered
    1. Go [here](https://73874bdaecee6ea6d7a506b37d5d256d--spectrum-web-components.netlify.app/review/)
    2. See the accepted VRT updates

## Types of changes

## Checklist
